src/common/platform/win32/i_main.cpp

```diff
@@ -560,10 +561,12 @@ INT_PTR CALLBACK ErrorPaneProc (HWND hDlg, UINT msg, WPARAM wParam, LPARAM lPara
 			if (LOWORD(wParam) == IDC_BUTTON1) // we pressed the restart button, so run GZDoom again
 			{
 				HMODULE hModule = GetModuleHandleW(NULL);
+				LPWSTR cmdline = (LPWSTR)calloc(wcslen(GetCommandLineW()) + sizeof("-waitforprocess") + 12, sizeof(WCHAR));
```

Comment:
Memory allocated by `calloc()` must be freed explicitly. That’s why usage of `TArray<>` is preferred.

Reply:
std::wstring would be preferred here. Please no C allocations unless there's no way around them, I'd rather migrate the entire code base off them.